academic/import_assets.py

```diff
@@ -37,51 +41,201 @@ def import_assets():
         log.error("Could not detect assets file. You may need to update Academic in order to use this tool.")
         return
 
+    params_filename = "config/_default/params.toml"
+    if not Path(params_filename).is_file():
+        log.error("Could not detect params.toml file.")
+        return
+
     # Create output dirs if necessary
     Path(JS_FILENAME).parent.mkdir(parents=True, exist_ok=True)
     Path(CSS_FILENAME).parent.mkdir(parents=True, exist_ok=True)
 
     # Parse TOML file which lists assets
-    parsed_toml = toml.load(assets_filename)
+    assets_toml = toml.load(assets_filename)
+
+    # Parse TOML file which lists assets
```

Comment:
Should be 'parameters' in the comment

Reply:
True, I have changed the comment do be more appropriate.